### PR TITLE
Remove the obsolete ServerKeyBits sshd_config option

### PR DESCRIPTION
**Pull request by smokris**
#244

### CODE REVIEW

**Comment by ghoneycutt** [comment]
Hi @smokris 

Thank you for your contribution. We can't just remove this option, because it is a valid option and could be in use and is used by default on some of the older platforms. If there are specific platforms in which it should not be used, such as EL7, I would be happy to merge changes to turn it off there.

File: manifests/init.pp [hunk 1]
```diff
@@ -143,7 +142,6 @@
       $default_sshd_gssapicleanupcredentials   = 'yes'
       $default_sshd_acceptenv                  = true
       $default_service_hasstatus               = true
-      $default_sshd_config_serverkeybits       = '1024'
```
Comment (ghoneycutt) [inline manifests/init.pp]: Needs a conditional here so that if EL7 then set to `undef`, else set to `'1024'`.

**Comment by tuxmea** [comment]
Fix released in https://github.com/ghoneycutt/puppet-module-ssh/pull/278 this one can be closed.

**Comment by ghoneycutt** [comment]
Fix is in v3.57.1

**Comment by EpiqSty** [comment]
Looks like that PR was'nt merged, but we are observing such errors on Debian 9 systems:

` rexec line 34: Deprecated option ServerKeyBits`

So, my question - is there any way to disable ServerKeyBits for given distribution via Hiera, or we have to hardcode every OS using the same way as it was done for RHEL 7.4 (#278)? 

@ghoneycutt , @tuxmea , @smokris

**Pull request closed without merging**